src/licensedcode/data/rules/fsfap-no-warranty-disclaimer_1.RULE

```diff
@@ -0,0 +1,15 @@
+---
+license_expression: fsfap-no-warranty-disclaimer
+is_license_text: yes
+minimum_coverage: 99
+ignorable_copyrights:
+    - Copyright (c) 2008 Micah J. Cowan
+ignorable_holders:
+    - Micah J. Cowan
+---
+
+Copyright (C) 2008 Micah J. Cowan
```

Comment:
Please do not add a rule with a copyright here. It adds nothing new... copyright will be detected separately ... here this is not a copyright for the license but the code.